datafusion/physical-plan/src/windows/bounded_window_agg_exec.rs

```diff
@@ -1262,4 +1642,111 @@ mod tests {
         assert_batches_eq!(expected, &batches);
         Ok(())
     }
+
+    #[tokio::test]
+    async fn bounded_window_exec_linear_mode_range_information() -> Result<()> {
```

Comment:
Add comment on test, why you add timeout and what do you expect if the test fail etc.